packages/cli/cloud/src/services/cli-api.ts

```diff
@@ -197,6 +207,23 @@ export async function cloudApiFactory(
       }
     },
 
+    async listEnvironments({ name }): Promise<AxiosResponse<ListEnvironmentsResponse>> {
+      try {
+        const response = await axiosCloudAPI.get(`/projects/${name}/environments`);
+
+        if (response.status !== 200) {
+          throw new Error('Error fetching cloud environments from the server.');
+        }
+
+        return response;
+      } catch (error) {
+        logger.debug(
+          "🥲 Oops! Couldn't retrieve your project's environments from the server. Please try again."
+        );
```

Comment:
Debug logs are for debugging, not displaying to the user there was an error. Could be great to add more technical information in this log, like error message, API endpoint, name.